src/server/life/MapleMonster.java

```diff
@@ -532,10 +533,16 @@ private static double calcExperienceStandDevThreshold(List<Float> entryExpRatio,
     
     private void distributePlayerExperience(MapleCharacter chr, float exp, float partyBonusMod, int totalPartyLevel, boolean highestPartyDamager, boolean whiteExpGain, boolean hasPartySharers) {
         float playerExp = (YamlConfig.config.server.EXP_SPLIT_COMMON_MOD * chr.getLevel()) / totalPartyLevel;
+        int map = chr.getMapId();
         if (highestPartyDamager) playerExp += YamlConfig.config.server.EXP_SPLIT_MVP_MOD;
         
         playerExp *= exp;
         float bonusExp = partyBonusMod * playerExp;
+
+        if(CustomMapExperience.HasCustomExpValue(map))
+        {
+            playerExp *= CustomMapExperience.GetCustomExpMultiplier(map);
```

Comment:
The bonusExp is unaffected by the CustomMapExperience. This would make party play unrewarding in these maps. We should think about moving this block above line 540

Reply:
I originally thought the multiplier should only affect the base experience, but I'll move this for now